xcell/cls/data.py

```diff
@@ -88,24 +229,25 @@ def read_data(self, data_path):
         return data
 
     def get_tracers_used(self, wsp=False):
-        tracers = []
-        for trk, trv in self.data['cls'].items():
-            tr1, tr2 = trk.split('-')
-            if trv['compute'] != 'None':
-                tracers.append(tr1)
-                tracers.append(tr2)
+        lab = ['no_wsp', 'wsp'][wsp]
+        if self.tracers[lab] is None:
+            tr_names = self._get_tracers_defined()
```

Comment:
Comment: first get all tracers in yaml file

Reply:
Done